README.md

```diff
@@ -70,7 +70,7 @@ The build steps are the same (`./configure; make; sudo make install`).
       -D, --describe-stats   : print stats description and exit
       -v, --verbose=N        : set logging level (default: 5, min: 0, max: 11)
       -o, --output=S         : set logging file (default: stderr)
-      -c, --conf-file=S      : set configuration file (default: conf/nutcracker.yml)
+      -c, --conf-file=S      : set configuration file (default: /etc/nutcracker/nutcracker.yml)
```

Comment:
This is backwards incompatible, only works on `*nix`-like systems, and is unrelated to new redis command support. Remove it.

Reply:
noted, thanks. I will revert it.